.pa11yci.js

```diff
@@ -0,0 +1,15 @@
+var URL_ENV = process.env.URL_ENV;
+var URL;
+
+if(URL_ENV == "Local"){
+    URL = "http://localhost:3000";
+} else {
+    URL = URL_ENV;
+}
```

Comment:
if URL_ENV - use that, otherwise, localhost. This logic is fine

IF you wanted to use an ENVIRONMENT type plag, I would expect DEVELOPMENT vs PRODUCTION type properties and then you'd say here if URL_ENV == DEVELOPMENT -> use the aws stuff, otherwise localhost.